File: CommonDomain/Sale.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CommonDomain
{
    [Serializable]
    public class Sale : Entity<int>
    {
        private string _buyerName;
        private int _ticketsBought;
        private int _showId;

        public Sale(string name, int tickets, int showid)
        {
            this._buyerName = name;
            this._ticketsBought = tickets;
            this._showId = showid;
        }

        public string GetName()
        {
            return this._buyerName;
        }

        public int GetTicketsBought()
        {
            return this._ticketsBought;
        }

        public int GetShowId()
        {
            return this._showId;
        }

        public void SetName(string newName)
        {
            this._buyerName = newName;
        }

        public void SetTicketsBought(int newTicketsBought)
        {
            this._ticketsBought = newTicketsBought;
        }

        public void SetShowId(int newShowId)
        {
            this._showId = newShowId;
        }

    }
}

File: CommonDomain/src/main/java/domain/Sale.java
package domain;

import java.io.Serializable;

public class Sale extends Entity<Integer> {
    String buyerName;
    Integer ticketsBought;
    Integer showID;


    public Sale(String buyerName, Integer ticketsBought, Integer showID) {
        this.buyerName = buyerName;
        this.ticketsBought = ticketsBought;
        this.showID = showID;
    }

    public String getBuyerName() {
        return buyerName;
    }

    public void setBuyerName(String buyerName) {
        this.buyerName = buyerName;
    }

    public Integer getTicketsBought() {
        return ticketsBought;
    }

    public void setTicketsBought(Integer ticketsBought) {
        this.ticketsBought = ticketsBought;
    }

    public Integer getShowID() {
        return showID;
    }

    public void setShowID(Integer showID) {
        this.showID = showID;
    }
}